library/src/main/java/com/mux/video/upload/internal/UploadInfo.kt

```diff
@@ -17,8 +17,11 @@ import java.io.File
  * Job and Flows populated
  */
 internal data class UploadInfo(
+  @JvmSynthetic internal var shouldStandardize: Boolean = false,
+  @JvmSynthetic internal var inputFileFormat: String? = "",
   @JvmSynthetic internal val remoteUri: Uri,
-  @JvmSynthetic internal val file: File,
+  @JvmSynthetic internal var file: File,
```

Comment:
Fwiw I ended up removing the iOS equivalent of this `file` property from `UploadInfo`